plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -924,6 +924,7 @@ items:
     - delete
     - get
     - list
+    - update
```

Comment:
what this for?

Reply:
This is so that the unit tests pass after enabling pod update capability in scheduler bootstrap policy